app/js/effectsTable/criterionCardDirective.html

```diff
@@ -114,29 +114,29 @@ <h5 id="criterion-title-{{ criterion.id }}">{{criterion.title}}</h5>
                 </div>
               </td>
 
-              <td id="soe-unc-{{ criterion.id }}-{{ dataSource.id }}" ng-if="!isInput">
+              <td id="soe-unc-{{ idx }}-{{ $index }}" ng-if="!isInput">
```

Comment:
if there's two 'index' variables at least one needs to have a semantic name or it's confusing